keps/sig-node/4816-dra-prioritized-list/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: DRA Prioritized List
+kep-number: 4816
+authors:
+  - "@johnbelamaric"
+owning-sig: sig-node
```

Comment:
@mrunalp @alculquicondor @MaciekPytel I did this as a sig-node KEP because it's DRA. But it actually only touches API server and kube-scheduler and (indirectly via scheduler libs?) cluster autoscaler. Thoughts?

Reply:
@pohly thoughts? I guess it's technically a sig-node owned scheduler plugin that gets tweaked? I don't know the ownership tree here that well.

Reply:
The code probably will be under `k8s.io/dynamic-resource-allocation/structured`, which is under SIG node because all of `k8s.io/dynamic-resource-allocation` is. We could add an exception for the `structured` sub-directory (= separate OWNERS file) because that is mostly used by schedulers.

Conceptually this feature seems indeed closer to SIG Scheduling than SIG Node.

Reply:
SGTM to switch to SIG Scheduling as owning. Thanks!